eva-server/dockerconf/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM tomcat:9.0.5-jre8
+ADD eva.war /usr/local/tomcat/webapps/
```

Comment:
I thought we agreed to copy the WAR from the target folder? Or shall we leave it as it because it is more flexible?

Reply:
When we build the project, the docker files are copied to the 'target' directory, so when we build the docker image we will be using the WAR from the target folder.